cmd/cape/cmd/root.go

```diff
@@ -25,7 +25,10 @@ type UserError struct {
 }
 
 func (e UserError) Error() string {
-	return fmt.Sprintf("%s: %s", e.Msg, e.Err.Error())
+	if e.Err != nil {
```

Comment:
Should we allow the Err on an error type to be nil? 🤔

Reply:
I guess we can always make it wrap a custom error, but that seems a bit cumbersome since we have to split the message and error message.